lib/Driver/ToolChains.cpp

```diff
@@ -124,6 +124,7 @@ static void addCommonFrontendArgs(const ToolChain &TC,
 
   inputArgs.AddAllArgs(arguments, options::OPT_I);
   inputArgs.AddAllArgs(arguments, options::OPT_F);
+  inputArgs.AddAllArgs(arguments, options::OPT_Fsystem);
```

Comment:
I think it's better to put -Fsystem last, so 'user' search paths get precedence.